makefiles/targets.mk

```diff
@@ -36,9 +38,24 @@ build:
 sanity-checks:
 	which $(CC) &>/dev/null || { echo "C compiler ($(CC)) not installed. Aborting."; exit 1; }
 
-#- runs only the first time and after `make update`, so have "normal"
-#  (timestamp-checked) prerequisites here
-deps-common: sanity-checks $(NIMBLE_DIR)
+#- check if "make update" was executed
+warn-update:
+	if [[ -e $(UPDATE_TIMESTAMP) ]]; then \
+		if [[ $$(cat $(UPDATE_TIMESTAMP)) -ne $$($(GET_CURRENT_COMMIT_TIMESTAMP)) ]]; then \
+			echo -e "\n\"$$(basename "$(MAKE)") update\" was not executed for the current Git commit! The resulting build is unsupported.\n"; \
+		fi; \
+	fi
+
+#- check if we're enabling multiple jobs - https://stackoverflow.com/a/48865939
+warn-jobs:
```

Comment:
this one probably only needs to be done once (by placing/depending on a .warned-jobs in `build/`)

Reply:
Where do you see it multiple times?

Reply:
Oh, you mean once per user. There's this other PR that removes any automagic build parallelism, making this warning rather important: https://github.com/status-im/nimbus-eth2/pull/2095